test/enforcer.test.ts

```diff
@@ -697,3 +697,10 @@ test('TestEnforceExWithPriorityModel', async () => {
   testEnforceEx(e, 'bob', 'data2', 'read', [true, ['data2_allow_group', 'data2', 'read', 'allow']]);
   testEnforceEx(e, 'alice', 'data2', 'read', [false, []]);
 });
+
```

Comment:
This should test all the normal test cases for this model + policy and verify all the other ramifications with an alternative matcher. Single test enforce can be flagged true or false out of the randomness of a faulty pull request.

Reply:
Could you review this PR once?